test/react/class-components/multiple-inheritance.js

```diff
@@ -0,0 +1,43 @@
+const React = require("react");
+this['React'] = React;
+
+class Parent extends React.Component {
+  constructor() {
+    super();
+    this.state ={
+      a: 100,
+    };
+  }
+  render() {
+    if (this.props.x === 5) {
+      return <span>{this.state.a}</span>;
+    } else {
+      return <span>Hello world</span>;
+    }
+  }
+}
+
+this['Adapter'] = Adapter;
+
+class Child extends Parent {
+  constructor() {
+    super();
+  }
+}
+
+class App extends React.Component {
+  render() {
+    return <div><Child x={10} /></div>;
+  }
+}
+
+App.getTrials = function(renderer, Root) {
+  renderer.update(<Root />);
+  return [['multiple inheritance', renderer.toJSON()]];
```

Comment:
Nit: this is just an inheritance chain, isn’t it? “Multiple inheritance” typically implies something different

Reply:
I've always caused this multiple inheritance as per typical OO terminology – maybe I have it wrong? I can change if so.